packages/shared/components/Notification/Notification.tsx

```diff
@@ -19,8 +19,6 @@ import styled, { css, useTheme } from 'styled-components';
 import { ButtonIcon, Flex, Text } from 'design';
 import { Close } from 'design/Icon';
 
-import type { propTypes } from 'design/system';
```

Comment:
`propTypes` does not include types for props used by our components, but rather [React PropTypes used by styled-system](https://github.com/styled-system/styled-system/blob/v3.1.11/src/index.js#L6-L16). `design/system` itself is not typed as of yet anyway.

Instead, I opted for defining the types for `Notification` in such a way that there's a bunch of props that are required but they also allow passing any extra `styleProps` to `Container`.